.github/workflows/ci-workflow.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       run: make -f Bootstrap.mak ${{ matrix.osname }} CONFIG=${{ matrix.config }}
     - name: Test
       run: bin/${{ matrix.config }}/premake5 test
+    - uses: actions/upload-artifact@v2
```

Comment:
It would be nice if this step had a better name, such as "Upload Artifacts". Same for the Windows step.